Assignments/Podcast_reflections/Question_ep3_pitfalls.md

```diff
@@ -0,0 +1,49 @@
+# Open Science for Physicists
+
+Especially for the course, we have recorded 4 podcast episodes extending the discussions of the course topics. 
+Participants are asked to listen to the podcast and answer some related questions. 
+
+
+## Questions for Episode 3: Pitfalls of Open Science Policies
+
+1. What embodies Inge’s definition of open science the most? 
+	+ Transparency & sharing resources (data, code, procedures, etc.).
+	+ Open access to the outcomes of research.
+	+ Commitment to eliminate fraudulent practices in research.
+2. Why was Inge not so enthusiastic about open access at first? She is worried that...
+	+ ... it costs a lot of money to publish. 
+	+ ... nonprofessionals can access the research and might misinterpret it.
+	+ ... it incentivizes publishers to publish low quality work since publishing more means collecting more publishing fees from authors.
+3. What is __not__ a concern that Inge has for openly sharing research and data with everyone (both scientists and general public)?
+	+ It is not useful to share if people do not understand how to use the data or research.
+	+ It can be dangerous to share research with the general public that has not been verified extensively yet.
+	+ If you share data openly, others might take advantage of your work without giving proper credits.
+4. What was the new primary outcome of Inge’s Parkinson study after consulting with Parkinson patients?
+	+ Shaking
+	+ Memory loss
+	+ Tiredness
+	+ Loss of orientation
+5. Inge states that there is currently no evidence that open science improves the quality of science.
+	+ True
+	+ False
+6. What does Inge mean that there is no evidence for or against the benefits of open science on quality of research and career paths?
+	+ She does not mention further details about this.
+	+ She means that data has been collected and it shows no evidence of benefits.
+	+ She means that the studies done on the benefits of open science do not follow the proper methods.
+	+ She means that there is a lack of studies on this specific question.
+7. What method does Inge advocate using to study the benefits of open science?
+	+ Interviewing researchers about how open science improved their research.
+	+ Analyzing quality of research from published studies with and without open science practices.
+	+ Doing a randomized controlled experiment between research with and without open science practices.
+8. Open science and recognition & rewards are similar concepts.
+	+ True
+	+ False
+9. What outcomes could one evaluate to assess the benefits of open science?
+	+ Reproducibility
+	+ Transparency
+	+ Both of the above
+	+ None of the above
+10. Does engaging in open science practices require creating new jobs in academia?
```

Comment:
I am not sure I understand this formulation of the question. Engaging by who? I don't exactly remember the context of the question, but in this form it reads like a rhetorical question

Reply:
I am not able to listen to the podcast now, but I think the point was that proper implementation of open science requires new jobs in academia. We could rephrase it to "Do open science practices require creating new jobs in academia?", or we can take the question out?

Reply:
I have tried different formulations myself but it always becomes a bit too simplistic. There are certain things that can be put in every person's work routine and other things that will require specialization. I find this topic always mixed with a bit of non-sequitur